.rubocop.yml

```diff
@@ -22,21 +26,19 @@ Metrics/LineLength:
 Metrics/ParameterLists:
   CountKeywordArgs: false
 
-# Indent one level for follow-up lines
-Style/MultilineOperationIndentation:
-  EnforcedStyle: indented
-
-# Allow small arrays of words with quotes
-Style/WordArray:
-  MinSize: 3
+Style/AccessorMethodName:
+  Exclude:
+    - 'lib/reek/context/visibility_tracker.rb'
 
-# Allow single-line method definitions
-Style/SingleLineMethods:
+# TODO: Discuss
```

Comment:
To expand on this a bit further, RuboCop started out enforcing `alias_method` everywhere, and this is how things stand in Reek at the moment. With the new version it changed to requiring `alias_method` inside methods, and `alias` outside. I have a slight preference for the new style, but I think we pick a style and have RuboCop enforce it either way.